src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/Order.sql

```diff
@@ -16,9 +16,12 @@
     Created DATETIME2 NOT NULL,
     LastUpdated DATETIME2 NOT NULL,
     LastUpdatedBy UNIQUEIDENTIFIER NOT NULL,
-    LastUpdatedByName NVARCHAR(256) NULL,     
-    CONSTRAINT FK_Order_OrderStatusId_OrderStatus_OrderStatusId FOREIGN KEY (OrderStatusId) REFERENCES OrderStatus (OrderStatusId), 
-    CONSTRAINT FK_Order_OrganisationAddressId_Address_AddressId FOREIGN KEY (OrganisationAddressId) REFERENCES Address (AddressId),
-    CONSTRAINT FK_Order_OrganisationBillingAddressId_Address_AddressId FOREIGN KEY (OrganisationBillingAddressId) REFERENCES Address (AddressId),
-    CONSTRAINT FK_Order_OrganisationContactId_Contact_ContactId FOREIGN KEY (OrganisationContactId) REFERENCES Contact (ContactId)
+    LastUpdatedByName NVARCHAR(256) NULL,   
+    
+    CONSTRAINT FK_Order_OrderStatus FOREIGN KEY (OrderStatusId) REFERENCES OrderStatus (OrderStatusId),    
+    CONSTRAINT FK_Order_OrganisationAddress FOREIGN KEY (OrganisationAddressId) REFERENCES [Address] (AddressId),
+    CONSTRAINT FK_Order_OrganisationBillingAddress FOREIGN KEY (OrganisationBillingAddressId) REFERENCES [Address] (AddressId),     
+    CONSTRAINT FK_Order_OrganisationContact FOREIGN KEY (OrganisationContactId) REFERENCES Contact (ContactId), 
+    CONSTRAINT FK_Order_SupplierAddress FOREIGN KEY (SupplierAddressId) REFERENCES [Address] (AddressId),     
+    CONSTRAINT FK_Order_SupplierContact FOREIGN KEY (SupplierContactId) REFERENCES Contact (ContactId)
 );

```

Comment:
This isn't essential (and is just me being exceptionally picky!) but I think there's a bit of trailing white space on a few lines here.

Reply:
👍